server/controllers/visitation.controllers.js

```diff
@@ -1,23 +1,44 @@
-import scheduleVisitation from '../services/visitation.service';
-// import { sendMail } from '../services/mailer.service';
-// import EMAIL_CONTENT from '../../mailer';
+import dotenv from 'dotenv';
+import { scheduleVisitation, getAllVisitations } from '../services/visitation.service';
+import EMAIL_CONTENT from '../../mailer';
+import { sendMail } from '../services/mailer.service';
 import httpStatus from '../helpers/httpStatus';
 
+dotenv.config();
+
+const { BALLERS_EMAIL } = process.env;
```

Comment:
Lets add a default value here to prevent the code from breaking if BALLERS_EMAIL is not set.
```suggestion
const { BALLERS_EMAIL } = process.env || 'dev@highrachy.com';
```